notebook/agentchat_sql.ipynb

```diff
@@ -0,0 +1,858 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f384223",
+   "metadata": {},
+   "source": [
+    "# SQL Agent\n",
```

Comment:
I would specifically call this agent a `SQL Lite Agent` cause it doesn't seem it can work with any other DB